backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/dialog/BpkDialog.kt

```diff
@@ -113,7 +118,7 @@ fun BpkWarningDialog(
     confirmButton: DialogButton,
     secondaryButton: DialogButton,
     linkButton: DialogButton? = null,
-    properties: DialogProperties = DialogProperties(),
+    properties: DialogProperties = DialogProperties(usePlatformDefaultWidth = false),
```

Comment:
I wonder if rather than exposing the `DialogProperties` directly we should just expose these two properties - I can't see others being configured:
``` val dismissOnBackPress: Boolean = true,
    val dismissOnClickOutside: Boolean = true,
```
otherwise users may remove the `usePlatformDefaultWidth` prop by accident